src/lib/eos/eos-account.ts

```diff
@@ -69,11 +69,6 @@ const buildNewAccountTransaction = async (
     .toDate()
     .toString()
   const sortedActiveKeys = [userKey, backupKey, serviceKey].sort()
-  if (sortedActiveKeys.length < 3) {
```

Comment:
Dobrze byłoby dodać to sprawdzenie. Najpierw napisałbym test i zobaczył czy sama libka to weryfikuje. Jeżeli nie, to samemu bym to napisał